packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -54,61 +56,13 @@ NS_ASSUME_NONNULL_BEGIN
                                                          (const facebook::react::ObjCTurboModule::InitParams &)params
  *   - (id<RCTTurboModule>)getModuleInstanceFromClass:(Class)moduleClass
  */
-@interface RCTAppDelegate : UIResponder <UIApplicationDelegate, UISceneDelegate, RCTBridgeDelegate>
+@interface RCTAppDelegate : UIResponder <UIApplicationDelegate, UISceneDelegate, RCTRootViewFactoryDelegate>
 
 /// The window object, used to render the UViewControllers
-@property (nonatomic, strong, nonnull) UIWindow *window;
-@property (nonatomic, strong, nullable) RCTBridge *bridge;
-@property (nonatomic, strong, nullable) NSString *moduleName;
-@property (nonatomic, strong, nullable) NSDictionary *initialProps;
-
-/**
- * It creates a `RCTBridge` using a delegate and some launch options.
- * By default, it is invoked passing `self` as a delegate.
- * You can override this function to customize the logic that creates the RCTBridge
- *
- * @parameter: delegate - an object that implements the `RCTBridgeDelegate` protocol.
- * @parameter: launchOptions - a dictionary with a set of options.
- *
- * @returns: a newly created instance of RCTBridge.
- */
-- (RCTBridge *)createBridgeWithDelegate:(id<RCTBridgeDelegate>)delegate launchOptions:(NSDictionary *)launchOptions;
```

Comment:
Removing these methods will be breaking.
Is there a way to keep them and forward them to the factory?

Reply:
I moved those methods back to `RCTAppDelegate.h` we now have the duplication of two methods: 

- `createBridgeWithDelegate`
- `createRootViewWithBridge` 

But I guess those two will be removed in over the time of next few releases (when new arch will be the default)